### DISCUSSION TITLE
Navigation to any vertices of a multipoint geometry. How it works in QField 3.3?

### DISCUSSION THREAD

**Discussion by pzsolt** [opengisch/QField#5370] [Q&A]
In previous versions of QField, navigation to individual vertices of multipoint shapes worked well. You could walk all the way through the vertices by the left/right arrows on the purple feature bar.
Now, in QField 3.3 it's changed. After setting a navigation destination using a feature, this purple bar is not appearing. How does this work in the new version? I can't find any information about this.
Please help me.

**Comment by pzsolt** [comment]
I found the vertices control arrows and the counter, but I think that the white arrows on the white background is not a very user-friendly solution, as is the unreadable counter. Please change this.

**Reply by nirvn** [reply]
Fixed here: https://github.com/opengisch/QField/pull/5378
